tests/pipelines/eks/awscli-cl2-load.yaml

```diff
@@ -14,7 +14,18 @@ spec:
   - name: nodes-per-namespace
   - name: cl2-load-test-throughput
   - name: results-bucket
+  - name: slack-hook
+  - name: slack-message
```

Comment:
Do we need to pass message as a param to pipeline? can we construct this message in the pipeline and send as a param to underlying task? The reason is we are accepting one message string in the pipeline, how do we differentiate if this is for test getting started / deleted?

Reply:
> Do we need to pass message as a param to pipeline? can we construct this message in the pipeline and send as a param to underlying task? The reason is we are accepting one message string in the pipeline, how do we differentiate if this is for test getting started / deleted?

Message here is going to be the job-name (1k/5k/10/15k/some other test name) that we want to post. We will prepend and append text to it in the tasks based on where we are using it like `<Job-name> got kicked off`. Not the complete message itself. 
Task/pipeline itself will be unaware of the meaningful job name itself that we want to see in slack notifications.